config.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+config = {
+    'SQLALCHEMY_DATABASE_URI': os.environ.get('DATABASE_URL'),
+    'SQLALCHEMY_TRACK_MODIFICATIONS':False,
+    'ACCESS_TOKEN_SECRET':os.environ.get('ACCESS_TOKEN_SECRET'),
+    'REFRESH_TOKEN_SECRET':os.environ.get('REFRESH_TOKEN_SECRET'),
+    'SENDGRID_EMAIL':os.environ.get('SENDGRID_EMAIL'),
+    'SENDGRID_API_KEY':os.environ.get('SENDGRID_API_KEY'),
+    'SENDGRID_SERVER_LINK':os.environ.get('SENDGRID_SERVER_LINK'),
+    'DEBUG': os.environ.get('DEBUG'),
+}
+
+for i in config.values():
```

Comment:
would recommend check directly within os.environ, why check in config object that you've created. Not mandatory, but cleaner that way